hrui/js/src/hrui.js

```diff
@@ -214,6 +182,37 @@
     $line.find('label').attr('for', $line.find('.crm-form-date').attr('id'));
   }
 
+  /**
+   * Manipulates, at the DOM level, the blocks/fields belonging to the
+   * Inline Custom Data custom fields set
+   */
+  function manipulateDOMOfInlineCustomData () {
+    // In Personal Details Page
+    var $inlineDataBlock = $('.Inline_Custom_Data');
+
+    if ($inlineDataBlock.length) {
+      $inlineDataBlock
```

Comment:
Is it possible to move this block inside its own function so we can explain what's going on?

For example:

```js
function manipulateDOMOfInlineCustomData () {
  // ...

  if ($inlineDataBlock.length) {
    doSomethingPrettyWithTheInlineCustomData(); // I have no idea what this does....
  }

  if ($('#customFields').length < 1) {
    wrapEachCustomDataLabelInsideATd(); // for example
  }
}
```